variables.tf

```diff
@@ -34,6 +34,18 @@ variable "log_analytics_workspace_enabled" {
   default     = true
 }
 
+variable "log_analytics_workspace_name_override" {
```

Comment:
Please make it two variables. `log_analytics_workspace_enable_override` and `log_analytics_workspace_override_name`. Please check platform-features module

Reply:
Please also apply a condition for checking the following regexp in the overriden_name. `The workspace name should include 4-63 letters, digits or '-'. The '-' shouldn't be the first or the last symbol.`